src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -11,15 +11,15 @@ public static class ImageEstimatorsCatalog
     {
         /// <include file='doc.xml' path='doc/members/member[@name="ImageGrayscalingEstimator"]/*' />
         /// <param name="catalog">The transform's catalog.</param>
-        /// <param name="columnPairs">The name of the columns containing the name of the resulting output column (first item of the tuple), and the paths of the images to work on (second item of the tuple).</param>
+        /// <param name="outputInputColumnPair">Names of the resulting output column (first item of the tuple), and the column containing the paths of the images to work on (second item of the tuple).</param>
```

Comment:
>outputInputColumnPair [](start = 25, length = 21)

they are not column pairs anymore. This might be leftover from when we wer eusing tuples.

Reply:
@sfilipi I see output-input tuples being used in the samples for several image transforms, despite the api signature accepting `params ColumnOptions[]`. 

For example, in the [`ConvertToGrayScale.cs`](https://github.com/dotnet/machinelearning/blob/178827f4e7a7ccd129a8ee60c96e9630ffc6c0b3/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToGrayScale.cs#L39-L40) sample, we have tuples being passed to both `LoadImages` and `ConvertToGrayscale`.
```csharp
            var pipeline = mlContext.Transforms.LoadImages(imagesFolder, ("ImageObject", "ImagePath"))
                           .Append(mlContext.Transforms.ConvertToGrayscale(("Grayscale", "ImageObject")));
```

Questions for you:
1. Are the tuples from the samples being cast to array or are the samples incorrect and need to be revised?
2. What name do you suggest for this?

Reply:
Not Senja, but maybe I can be useful as well.
1. ColumnOptions have implicit cast from tuples, so user can enter tuples if he wants to, but in general we work with array of ColumnsOptions. So examples are fine, at least for now.
2. ` /// <param name="columns">Specifies the names of the input columns for the transformation, and their respective output column names.</param>` I think this is our default choice for such cases.


---
In reply to: [263581917](https://github.com/dotnet/machinelearning/pull/2875#discussion_r263581917) [](ancestors = 263581917)

Reply:
renamed as suggested